src/transaction/submit.js

```diff
@@ -27,7 +27,10 @@ function formatResponse(response) {
   if (isImmediateRejection(response.engine_result)) {
     throw new utils.common.errors.RippledError('Submit failed');
   }
-  return convertKeysFromSnakeCaseToCamelCase(response);
+  return {
+    resultCode: response.engine_result,
+    resultMessage: response.engine_result_message
+  };
```

Comment:
This would mean that there will be no `tx_json` field in `submit` response.
Right now I have `tx_json`, which is very convenient - I can see what this transaction it was, and it's hash.